content/en/docs/How-to guides/clone-repository.md

```diff
@@ -0,0 +1,498 @@
+    https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+    ```
+
+    See the [Pipelines installation documentation][pipelines-inst] for other
+    installation options.
+
+1.  Install the [Tekton CLI, `tkn`][tkn-inst], on your machine.
+    
+[pipelines-inst]: /docs/pipelines/install/
+[tkn-inst]: /docs/cli/
+
+If this is your first time using Tekton Pipelines, we recommend that you
+complete the [Getting Started tutorials][getting-started] before proceeding with
+this guide.
+
+## Pull source code from git
+
+In this section you are going to create a Pipeline containing a Task to pull
+code from a git repository.
+
+### Create the Pipeline
+
+One practical aspect of Tekton Tasks and Pipelines is that they are reusable.
+There's a [community hub][tekton-hub] with a curated list of Pipelines and Tasks
+that you can include in your own CI/CD workflow. 
+
+This Pipeline references [git-clone][git-clone], a Task from the community hub.
+You have to download and apply this task to your cluster before running the
+Pipeline.
+
+Create `pipeline.yaml` with the following contents:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: clone-read
+spec:
+  description: | 
+    This pipeline clones a git repo, then echoes the README file to the stout.
+  params:
+  - name: repo-url
+    type: string
+    description: The git repo URL to clone from.
+  workspaces:
+  - name: shared-data
+    description: | 
+      This workspace contains the cloned repo files, so they can be read by the
+      next task.
+  tasks:
+  - name: fetch-source
+    taskRef:
+      name: git-clone
+    workspaces:
+    - name: output
+      workspace: shared-data
+    params:
+    - name: url
+      value: $(params.repo-url)
+```
+
+This code creates a new pipeline named `clone-read`. These are the most relevant
+values:
+
+
+- `params`: A list of parameters this Pipeline requires.
+  - `repo-url`: This parameter is the URL the task is going to pull the source code
+    from. 
+
+- `workspaces`: A list of file system paths to be defined by a TaskRun or a
+  PipelineRun. This allows a Task to set a mount point for configurations or to
+  share data with other Tasks. See the [Workspaces documentation][workspaces] to
+  learn more.
+  - `shared-data`: The name of the Workspace to store the source code.
+
+- `tasks`: A list of Tasks to be run sequentially. For now, there's only one
+  task: `fetch-source`. The field `taskRef: git-clone` is a reference to a
+  Task from the Tekton community hub. Later in this tutorial we are going to
+  install the Task on the cluster to have it available at run time.
+  - `workspaces`: Workspaces required by this Task. These Workspaces are
+    specific for this task and may be different from the ones defined at
+    Pipeline level.
+    - `source`: The Workspace `shared-data` is passed from the Workspaces
+      declared at Pipeline-level. This prevents name-clashing when two or more
```

Comment:
I got this from a Workshop. At least two attendants asked about duplicate names and how to avoid them.

The how-tos in this section are intended for a somewhat more experienced audience, who are at least comfortable using Kubernetes. Based on that I would argue that they probably understand and care about this.